cumulusci/cumulusci.yml

```diff
@@ -11,6 +11,12 @@ tasks:
         class_path: cumulusci.tasks.metadata_etl.AddRelatedLists
         options:
             namespace_inject: $project_config.project__package__namespace
+    add_page_layout_fields:
+        group: Metadata Transformations
+        description: Adds specified Fields to one Page Layout.
+        class_path: cumulusci.tasks.metadata_etl.AddFields
+        options:
+            namespace_inject: $project_config.project__package__namespace
```

Comment:
@BrianKSFDO We have a pattern for inferring this option now, so it's okay to leave it out of the task definition (more below).

Reply:
I'll remove `namespace_inject` in my next commit 👍